src/main/java/io/github/movax10/calorizer/controller/FacebookController.java

```diff
@@ -0,0 +1,89 @@
+package io.github.movax10.calorizer.controller;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.Scanner;
+
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.InitializingBean;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.restfb.DefaultFacebookClient;
+import com.restfb.FacebookClient;
+import com.restfb.types.Account;
+import com.restfb.types.User;
+
+
+
+/**
+ * @author radchenko.as
+ * @version 1.0
+ */
+@Controller
+public class FacebookController extends AbstractController implements
+		InitializingBean {
+
+	@Value("${facebook.clientId}")

```

Comment:
Please move all injected values to separate class like in http://tuhrig.de/why-using-springs-value-annotation-is-bad/. Then please move all static strings like "https://graph.facebook.com/oauth/authorize?client_id="
+				+ facebookClientId + "&redirect_uri=http://" + applicationHost
+				+ "/fromfb" + "&scope=email,user_location,user_birthday"; to variables